jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/s2i-core-hooks

```diff
@@ -8,6 +8,9 @@ function s2i_core_env_init_hook() {
 
 # add support for copying modules
 function s2i_core_copy_artifacts_hook() {
+  # At this point the settings are no more valid to start a server or use CLI.
+  # Replace s2i with startup one. Required by extension scripts.
+  replace_default_settings_with_startup
```

Comment:
In case one call CLI during s2i phase (not in an extension at startup). The maven settings need to be correct. With slim server, artifacts are retrieved from maven for both server and CLI (any JBoss module based app).